src/app/shared/access-control-form-container/access-control-form-container.component.spec.ts

```diff
@@ -0,0 +1,38 @@
+//
+// describe('AccessControlFormContainerComponent', () => {
+//   let component: AccessControlFormContainerComponent<any>;
+//   let fixture: ComponentFixture<AccessControlFormContainerComponent<any>>;
+//
+//   let bulkAccessConfigDataServiceMock: BulkAccessConfigDataService;
+//
+//   beforeEach(async () => {
+//
+//     bulkAccessConfigDataServiceMock = jasmine.createSpyObj('BulkAccessConfigDataService', {
+//       findByName: jasmine.createSpy('findByName'),
+//     });
+//
+//
+//     await TestBed.configureTestingModule({
+//       declarations: [ AccessControlFormContainerComponent ],
+//       imports: [ CommonModule, ReactiveFormsModule, SharedBrowseByModule, TranslateModule, NgbDatepickerModule ],
+//       providers: [
+//         { provide: BulkAccessConfigDataService, useValue: bulkAccessConfigDataServiceMock },
+//       // private bulkAccessControlService: BulkAccessControlService,
+//       // private selectableListService: SelectableListService,
+//       // protected modalService: NgbModal,
+//       // private cdr: ChangeDetectorRef
+//       ]
+//     })
+//     .compileComponents();
+//   });
+//
+//   beforeEach(() => {
+//     fixture = TestBed.createComponent(AccessControlFormContainerComponent);
+//     component = fixture.componentInstance;
+//     fixture.detectChanges();
+//   });
+//
+//   it('should create', () => {
+//     expect(component).toBeTruthy();
+//   });
+// });
```

Comment:
It would be good to add some tests for this component, as it isn't trivial

Reply:
This component is still missing tests as @artlowel initially noted two weeks ago.